myuw/templates/handlebars/card/international_student.html

```diff
@@ -0,0 +1,11 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "international_student_card_content" %}
+<div class="myuw-card">
+    <h3 class="myuw-card-title">
+        International Student
+    </h3>
+
+    <div class="myuw-card-section last">
```

Comment:
Looks like this card is still empty. Will approve the PR when this is finished.

Reply:
just an empty template for now, not live. this card has been comment out on landing.js.

Reply:
ah ok, sounds good!